torch/onnx/_internal/fx/export.py

```diff
@@ -33,6 +36,70 @@ def _onnx_not_available(name):
 from torch.utils import _pytree
 
 
+class ModuleExpansionTracer(torch.fx._symbolic_trace.Tracer):
+    """Tracer to create ONNX-exporting friendly FX graph.
+
+    This tracer traces models into operators. That is,
+    the traced graph mostly contains call_function nodes and
+    has no call_module nodes. The call_module nodes
+    are problematic to the use of make_fx(...) in ONNX
```

Comment:
> The call_module nodes are problematic to the use of make_fx(...) in ONNX exporter.

Could you elaborate? I'm wondering
* Is this a `make_fx` issue? Or is it specifically to ONNX exporter's use?
* Why are `call_module` problematic?
* Why are `call_module` nodes not lowered/decomposed by `make_fx`? (which I thought it would, since the resulting graph of `make_fx` is in 'aten/prim' IR)